torchao/_models/llama/eval.py

```diff
@@ -68,12 +68,17 @@ def run_evaluation(
             quantize_(model, int8_weight_only())
```

Comment:
btw, does eval work for you? I'm still getting the same result, e.g. for bfloat16 and int8wo, haven't tried other quantization types yet

Reply:
yeah all runs without issue
what error do you see? there might have been an update to lm_eval we need to addressw

Reply:
no errors, just getting the exact same eval number for bfloat16 and int8wo, I'll run more quant types tomorrow